lib/components/admin/field-trip-list.js

```diff
@@ -30,6 +39,14 @@ class FieldTripList extends Component {
     this.props.setActiveFieldTrip(null)
   }
 
+  _getReportUrl = () => {
+    const {datastoreUrl} = this.props
+    const {date} = this.state
+    const [year, month, day] = date.split('-')
+    const params = {month, day, year}
+    return `${datastoreUrl}/fieldtrip/opsReport?month=${qs.stringify(params)}`
```

Comment:
Is this a bug on my side?

![image](https://user-images.githubusercontent.com/56846598/123707137-40fc2780-d837-11eb-995a-815db845fa8f.png)

Reply:
No idea. But I do see that the month= needs to be removed from that URL string. Fixed in 72cd7de52e3bd5e38606f7fb7ffd3210ef034d3f